modules/vaos/app/docs/vaos/v0/vaos.yaml

```diff
@@ -1,3094 +0,0 @@
-openapi: "3.0.0"
```

Comment:
@cferris32 this also looks outdated to me as well so I removed it. Can you double check my understanding here?